src/Amqp/Examples/Amqp.V1.Sink/Amqp.V1.Sink.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Need to add <IsPackable>false</IsPackable> here or we're going to accidentally push a NuGet package. Dumb, I know, but I've done it before

Reply:
Done